contracts/PatriciaTree/Data.sol

```diff
@@ -47,7 +47,7 @@ library Data {
     if (pos == 0) {
       prefix.data = bytes32(0);
     } else {
-      prefix.data = bytes32(uint(self.data) & ~uint(1) << 255 - pos);
+      prefix.data = bytes32(uint(self.data) & ~uint(1) << 255 - pos); // ignore-swc-101
```

Comment:
Need to investigate how could the bitshift overflow in the 2 places in this contract?